go.mod

```diff
@@ -292,42 +288,42 @@ replace (
 	go.etcd.io/etcd/raft/v3 => github.com/openshift/etcd/raft/v3 v3.5.1-0.20220707134052-31b6b2d9b4d7 // from etcd
```

Comment:
You may want to make it explicit that we're pinning the etcd version here and why, by replacing the `// from etcd` with `// override <reason>`. Then the rebase script should leave this alone.

Reply:
Is that the case for other etcds in the replace? Can you check if change I made in `rebase.sh` is even legit? :D

Reply:
I don't think your change to `rebase.sh` is necessary if you used the override here in the go.mod.